### PR TITLE
Add datahub linkedin namespace to osc-cl1.

### PR DESCRIPTION
**Pull request by HumairAK**
Related: https://github.com/os-climate/os_c_data_commons/issues/80

### CODE REVIEW

File: cluster-scope/base/core/namespaces/datahub-linkedin/namespace.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+    name: datahub-linkedin
+    annotations:
+        openshift.io/requester: datahub-linkedin
+        openshift.io/display-name: "DataHub: The Metadata Platform for the Modern Data Stack"
+        op1st/project-owner: operate-first
+        op1st/onboarding-issue: "https://github.com/os-climate/os_c_data_commons/issues/807"
```
Comment (4n4nd) [inline cluster-scope/base/core/namespaces/datahub-linkedin/namespace.yaml:9]: issue link doesn't seem to work, maybe it should be: https://github.com/os-climate/os_c_data_commons/issues/163 ?

Comment (4n4nd) [inline cluster-scope/base/core/namespaces/datahub-linkedin/namespace.yaml]: issue link doesn't seem to work, maybe it should be: https://github.com/os-climate/os_c_data_commons/issues/163 ?

**Review by 4n4nd — CHANGES_REQUESTED** [review]

File: cluster-scope/base/core/namespaces/datahub-linkedin/namespace.yaml (hunk 1 above)
Comment (HumairAK) [reply cluster-scope/base/core/namespaces/datahub-linkedin/namespace.yaml]: fixed

**Review by 4n4nd — APPROVED** [review]
/lgtm

**Pull request merged**